python/servo/post_build_commands.py

```diff
@@ -103,8 +103,9 @@ def run(self, params, build_type: BuildType, android=None, debugger=False, debug
                 print("Android on-device debugging is not supported by mach yet. See")
                 print("https://github.com/servo/servo/wiki/Building-for-Android#debugging-on-device")
                 return
+            app_name = "org.servo.servoshell"
```

Comment:
I think it makes sense to move this to a global constant named something like `ANDROID_APP_NAME`.